larq/layers_test.py

```diff
@@ -101,6 +101,43 @@ def test_binarization(self, quantized_layer, layer, input_shape, kwargs):
 
         self.assertAllClose(quant_output, fp_output)
 
+    @parameterized.named_parameters(
+        (
+            "QuantDepthwiseConv2D",
+            lq.layers.QuantDepthwiseConv2D,
+            tf.keras.layers.DepthwiseConv2D,
+            (2, 3, 7, 6),
+            dict(kernel_size=3),
+        )
+    )
+    def test_depthwise_layers(self, quantized_layer, layer, input_shape, kwargs):
```

Comment:
I'd prefer if you either integrate this test into the existing layer tests, or remove the parameterization, since it only has one set of parameters anyway.